packages/kit/src/exports/public.d.ts

```diff
@@ -685,8 +686,8 @@ export interface KitConfig {
  */
 export type Handle = (input: {
 	event: RequestEvent;
-	resolve(event: RequestEvent, opts?: ResolveOptions): MaybePromise<Response>;
-}) => MaybePromise<Response>;
+	resolve(event: RequestEvent, opts?: ResolveOptions): MaybePromise<void | ResponseInit | Response>;
```

Comment:
Is returning nothing or a ResponseInit important to cancel the websocket connection?
I'm wondering if we can avoid this change so that it's not a breaking change.

Reply:
I do think there is a path to improving the dev experience here, we have an issues logged on the crossws repo to discuss further.

Reply:
Related to https://github.com/unjs/crossws/issues/88

Reply:
Same, would love to keep this as is 
https://github.com/unjs/crossws/pull/90